back/app/serializers/web_api/v1/project_serializer.rb

```diff
@@ -94,6 +94,8 @@ class WebApi::V1::ProjectSerializer < WebApi::V1::BaseSerializer
 
   has_many :project_images, serializer: WebApi::V1::ImageSerializer
   has_many :areas
+  has_many :topics, serializer: WebApi::V1::TopicSerializer
+  has_many :projects_topics
```

Comment:
Do we need to send `projects_topics` to the client? I'm wondering how client can use it.